mon-pix/app/routes/index.js

```diff
@@ -10,7 +10,7 @@ export default Route.extend(AuthenticatedRouteMixin, {
     const usesProfileV2 = this.currentUser.get('user.usesProfileV2');
 
     if (usesProfileV2) {
-      return this.transitionTo('profilv2');
+      return this.transitionTo('profile-v2');
```

Comment:
Pourquoi renommer cette route avec le `-` ?
Si ce n'est pas un besoin, je ne vois pas l'utilité de renommer une route que les utilisateurs ont enregistrés/ont en lien

Reply:
Alors l'endroit que pointe ta review c'est pas une route url d'un utilisateur mais une route Ember. Ceci dit c'est vrai que j'ai rajouté le '-' dans l'url aussi parce qu'une url est censée être en dash-case et pas en camelCase (ce qu'elle n'était même pas d'ailleurs), bref tant qu'à y être à renommer autant le faire ici. Quand on urls enregistrées ça ne pose pas de problème, ça fera un not-found qui les redirigera vers la page par défaut soit profil-v2, il ne le verront même pas.

Reply:
J'aimerais qu'on vérifie d'abord avec le métier pour cette route, sachant qu'elle a été souvent donnée "à la main" aux utilisateurs pour qu'ils puissent y accéder

Reply:
Overkill à mon avis vu qu'il existe déjà une redirection par défaut sur les urls inconnues qui redirige vers le profil.

Reply:
Je pense que le changement de route doit être repoussé après la migration